### ISSUE TITLE
https://training.github.com/ - Cheat Sheet - English (PDF) typo "gitgnore" in header

### ISSUE THREAD

**Issue by DougDelaney** [github/training-kit#810]
## Overview
**TL;DR**
"The .gitgnore file" heading has a typo in the [English (PDF)](https://training.github.com/downloads/github-git-cheat-sheet.pdf) cheatsheet.  It is fine in [English](https://training.github.com/downloads/github-git-cheat-sheet/) html.

### Extra Detail
**The .gitgnore file**
Sometimes it may be a good idea to exclude files from being
tracked with Git. This is typically done in a special file named
 .gitignore . You can find helpful templates for .gitignore
files at github.com/github/gitignore.

#### Screenshots
![image](https://user-images.githubusercontent.com/38562102/105944968-54118e00-6032-11eb-8527-79a50b950fc6.png)

**Comment by mikerobards** [comment]
Hello! I would like to correct this issue! Thank you!
